modules/sso/class.jetpack-sso-helpers.php

```diff
@@ -22,6 +24,12 @@ static function should_hide_login_form() {
 		return (bool) apply_filters( 'jetpack_remove_login_form', get_option( 'jetpack_sso_remove_login_form', false ) );
 	}
 
+	/**
+	 * Returns a boolean value for whether logging in by matching the WordPress.com user email to a
+	 * Jetpack site user's email is allowed.
```

Comment:
I am not sure if it is absolutely necessary but it wouldn't hurt to add @since 4.1 to the methods as well.